### PR TITLE
Update CComponent.php

### PR DESCRIPTION
**Pull request by devcommiter**
A little protection of the eval() usage in the evaluateExpression method.

### CODE REVIEW

**Comment by samdark** [comment]
Why closing?

File: framework/base/CComponent.php [hunk 1]
```diff
@@ -609,7 +609,14 @@ public function evaluateExpression($_expression_,$_data_=array())
 		if(is_string($_expression_))
 		{
 			extract($_data_);
-			return eval('return '.$_expression_.';');
+			try
+			{
+				return @eval('return ' . $_expression_ . ';');
```
Comment (cebe) [inline framework/base/CComponent.php]: why did you add the `@` here?

Comment (devcommiter) [inline framework/base/CComponent.php]: For suppressing errors from eval() function. I think in this case we will simply parse a necessary exception or get clear false without any additional messages. Similar solution you can see \CChoiceFormat::evaluate or in the \CAuthManager::executeBizRule or \CTimestampBehavior::getTimestampByAttribute

Comment (samdark) [inline framework/base/CComponent.php]: That changes behavior to be silent if PHP version is lower than 7. I don't think we should change it.

**Comment by samdark** [comment]
Looks good to me. Please add a line for changelog.

**Comment by devcommiter** [comment]
Hi, there is already exist changelog's comment.
`Bug #4130: Fixed PHP 7 added an interception of the ParseError exception for the eval() usage (devcommiter)`

**Comment by samdark** [comment]
Merged. Thanks!
